src/main/java/org/ladocuploader/app/FileExportController.java

```diff
@@ -149,8 +182,8 @@ private ResponseEntity handleCsvGeneration(String flow, String submissionId, Htt
                     .headers(headers)
                     .body(csvDoc.getCsvData());
         } else {
-            log.error("Attempted to download PDF with submission_id: " + submissionId + " but session_id was: "
-                    + httpSession.getAttribute("id"));
+            log.error("Attempted to download PDF with submission_id: " + submissionIdSanitized + " but session_id was: "
+                    + sanitizeSubmissionId(httpSession.getAttribute("id").toString()));
```

Comment:
What are we checking here with `httpSession.getAttribute("id")`? In the DataInterceptor, it looks like theres a flow-to-submissionID map that we keep the submission attributes in - is that related to this?

Reply:
@bseeger do you have the context here?

Reply:
Looks like it's trying to grab the submission id the old way, where as it needs to grab it from a flow-to-submissionId map as @sree-cfa suggests.    Not really sure what Lauren's goal was here, but since the `if` was for ensuring the submission id == the one in the session, I'm guessing she meant to pull the id from the map that Sree mentioned.

Reply:
Sorry I should have tagged @lkemperman-cfa here 😅

Reply:
Hi! That is not supposed to be there (should be fetching submission id the new way), but the sanitize method is to avoid log injection security issue.